sdk/keyvault/azure-security-keyvault-perf/pom.xml

```diff
@@ -21,6 +21,9 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <maven.compiler.source>1.8</maven.compiler.source>
     <maven.compiler.target>1.8</maven.compiler.target>
+    <codesnippet.skip>false</codesnippet.skip>
+    <javadocDoclet></javadocDoclet>
+    <javadocDocletOptions></javadocDocletOptions>
```

Comment:
Do we need this in this project or other perf projects if we are not going to make use of the codesnippets tool in it?

Reply:
Until the previous codesnippet tooling is replaced, yes as it will still attempt to run during Javadoc generation